src/init/features/apphosting/index.ts

```diff
@@ -212,9 +213,32 @@ export async function onboardRollout(
   const buildId = await apphosting.getNextRolloutId(projectId, location, backendId, 1);
   const buildOp = await apphosting.createBuild(projectId, location, backendId, buildId, buildInput);
 
-  const rolloutOp = await apphosting.createRollout(projectId, location, backendId, buildId, {
+  const rolloutBody = {
     build: `projects/${projectId}/locations/${location}/backends/${backendId}/builds/${buildId}`,
-  });
+  };
+
+  let tries = 0;
```

Comment:
This only gives us ~5 seconds of total delay (slightly more depending on how long CreateRollout takes). Is this enough? Perhaps we should consider more retries or adding exponential backoff

Reply:
It's been enough this far. Working on improving it on the backend as well.